application/common/helpers/Utils.php

```diff
@@ -0,0 +1,88 @@
+<?php
+namespace common\helpers;
+
+use yii\base\Security;
+
+class Utils
+{
+
+    const DT_Format = 'Y-m-d H:i:s';
+
+    public static function getDatetime($timestamp=false)
```

Comment:
Can we change `$timestamp=false` to `$timestamp = null` to match our use of null (not false) as our "no data" value?  (Similarly for `getIso8601(...)` below.)